dpgen/dispatcher/PBS.py

```diff
@@ -92,10 +97,9 @@ def sub_script_head(self, res):
         ret = ''
         ret += "#!/bin/bash -l\n"
         if res['numb_gpu'] == 0:
-            ret += '#PBS -l nodes=%d:ppn=%d\n' % (res['numb_node'], res['task_per_node'])
+            ret += '#PBS -l select=%d:ncpus=%d\n' % (res['numb_node'], res['task_per_node'])
         else :
-            ret += '#PBS -l nodes=%d:ppn=%d:gpus=%d\n' % (res['numb_node'], res['task_per_node'], res['numb_gpu'])
-        ret += '#PBS -l walltime=%s\n' % (res['time_limit'])
+            ret += '#PBS -l select=%d:ncpus=%d:ngpus=%d\n' % (res['numb_node'], res['task_per_node'], res['numb_gpu'])
```

Comment:
This modification will break on other PBS systems! considering provide a key to tell if the PBS is for ALI-EHPC